src/NServiceBus.Core/Headers.cs

```diff
@@ -58,6 +54,10 @@ public static class Headers
         /// <summary>
         /// Prefix included on the wire when sending applicative headers.
         /// </summary>
+        [ObsoleteEx(
```

Comment:
Shouldn't this be TreatAsErrorFromVersion = 8, RemoveInVersion = 9?

Reply:
I don't see why we would have to deprecate that property that way tbh. It's not supposed to be a user facing API where we need to drive them to a newer API. But happy to discuss whether we should remove it in v9 instead.

Reply:
yes, that was intended. Do you think it would be valuable to show a better obsolete message instead of just removing it instead?

Reply:
We usually keep things around long enough to warn, then error about it, but if you don't think that's needed here, then I'm fine with it.

Reply:
thoughts @Particular/nservicebus-maintainers ?

Reply:
Since its unlikely that the header is used by anyone I'm happy with just removing it